web/packages/teleport/src/components/TextSelectCopy/TextSelectCopyMulti.tsx

```diff
@@ -44,6 +49,16 @@ export function TextSelectCopyMulti({ lines, bash = true }: Props) {
     selectElementContent(targetEl as HTMLElement);
   }
 
+  function saveContentAsFile(content: string): void {
+    const a = document.createElement('a');
+    const blob = new Blob([content], { type: 'plain/text' });
+    a.href = window.URL.createObjectURL(blob);
+    a.download = saveContent.filename;
+    document.body.appendChild(a);
+    a.click();
+    document.body.removeChild(a);
+  }
```

Comment:
moved to shared utils in https://github.com/gravitational/teleport/pull/37232/commits/a5f8383b21b3af0b396a819f0f2df24caae648fc

However, I haven't updated the http file download (the code you referenced). FYI, this can also replace the save function that is used in RDP terminal. But I'll have to test that once with proper file transfer and given my current bandwidth, I would prefer to follow up to that later on. Let me know if you still think that should be done in this PR. 

> also can this throw an error at anytime? (do we need like a try catch block?)

No try catch needed here as all is DOM operation. I tried to add it just in case but my editor complained `useless try catch block` :D